### PR TITLE
DCOS-54811 - Make the DC/OS Terraform integration tests required checks.

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**
## High-level description

Make DC/OS Terraform integration tests required checks.

## Corresponding DC/OS tickets (required)

  - [DCOS-54811](https://jira.mesosphere.com/browse/DCOS-54811) - Make the DC/OS Terraform integration tests required checks.

## Related tickets (optional)

  - [DCOS-54316](https://jira.mesosphere.com/browse/DCOS-54316) - Enable dcos-terraform tests for integration test group 3.

### CODE REVIEW

File: mergebot-config.json [hunk 1]
```diff
@@ -52,8 +53,13 @@
                     "teamcity/dcos/test/aws/onprem/static/group2",
                     "teamcity/dcos/test/aws/onprem/static/group3",
                     "teamcity/dcos/test/aws/onprem/static/group4",
-                    "teamcity/dcos/test/test-e2e/group1"],
```
Comment (jbarrick-mesosphere) [inline mergebot-config.json]: I've changed the formatting to make this (and future) changes a little simpler.

**Comment by jbarrick-mesosphere** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jbarrick-mesosphere** [comment]
@mesosphere-mergebot request-review @orsenthil @cprovencher @pcrews

File: mergebot-config.json (hunk 1 above)
Comment (orsenthil) [reply mergebot-config.json]: Thanks!  sgtm.

**Comment by jbarrick-mesosphere** [comment]
@mesosphere-mergebot changelog-not-required test changes

**Pull request merged**